chips/sam4l/src/spi.rs

```diff
@@ -233,7 +232,7 @@ impl spi::SpiMaster for Spi {
         let regs: &mut SpiRegisters = unsafe { mem::transmute(self.registers) };
 
         self.enable_clock();
-
+        self.set_baud_rate(40000);
```

Comment:
so should applications always have to set the spi baud rate, and not have it inited to 40khz?